src/api/hooks/useGetANS.ts

```diff
@@ -123,6 +123,8 @@ export const knownAddresses: Record<string, string> = {
   "0xcd7b88c2181881bf8e7ef741cae867aee038e75df94224496a4a81627edf7f65": "Defy",
   "0xa3111961a31597ca770c60be02fc9f72bdee663f563e45223e79793557eef0d9":
     "Lucid Finance",
+  "0xd47ead75b923422f7967257259e7a298f029da9e5484dc7aa1a9efbd4c3ae648":
+    "USDt Redemption",
```

Comment:
Can we rename this to native?
```suggestion
  "0xd47ead75b923422f7967257259e7a298f029da9e5484dc7aa1a9efbd4c3ae648":
    "Native FA Redemption",
```